src/Shared/Components/Backdrop/backdrop.scss

```diff
@@ -0,0 +1,5 @@
+.backdrop {
+  background: #000000bf;
```

Comment:
Check if something is present in base for rgba

Reply:
not present